packages/ckeditor5-build-helpjuice-v2/src/hjeditor.ts

```diff
@@ -349,6 +349,18 @@ HelpjuiceEditor.defaultConfig = {
 						`<video controls="" style="max-width: 100%" src="${getUrl}">Your browser does not support HTML5 video.</video>`
 					);
 				}
+			},
+			{
+				name: 'loomProvider',
+				url: new RegExp(`^https:\\/\\/www\\.loom\\.com\\/share\\/(\\w+)`),
+				html: match => {
+					const getUrl = match.input;
+					const videoId = getUrl?.split('/').pop();
```

Comment:
```suggestion
					const videoId = match.input?.split('/').pop();
```

Reply:
I found that I can use `const videoId = match[ 1 ];`

![Screenshot from 2023-08-16 23-12-49](https://github.com/Helpjuice/ckeditor5/assets/45231276/51642937-2249-4aa4-a35a-ffd6398e3cb0)